css.md

```diff
@@ -0,0 +1,32 @@
+# CSS
+
+**Prerequisite knowledge**: basic html elements
+
+## Intro
+CSS (Cascading Style Sheets) is a style sheet language used for describing the presentation of document written in a markup language (like HTML).
+	CSS can improve content accessibility and readability by affecting layout, colors and fonts. Also  provide more flexibility and control in the specification of presentation characteristics.
+	Enable multiple web pages to share formatting by specifying the relevant CSS in a separate .css file, and reduce complexity and repetition in the structural content. 
+	The name cascading comes from the specified priority scheme to determine which style rule applies if more than one rule matches a particular element. 
+	This cascading priority scheme is predictable. [source](https://en.wikipedia.org/wiki/Cascading_Style_Sheets)
+
+## Classification / acceptance questions
+
+### Apprentice aka _Short skillset description, example: "I can branch!", "My First TODO app"_
+	*I know what is a box model, and I know its basic elements (graphical interpretation)
+	*I know how html elements could be displayed (in line, in line block, flex, table)
+	*I know what is the difference between in line, in line block and flex
+	*I know whats are classes
+	*I know how inheritance works
+	*I can change colors and fonts of text
+	*I can use padding, margin, and border attributes
+	*I can modify elements by ID
+	*I know the difference between basic selectors (*, #X and .X)
+### Intermediate aka _..._
+	*I can make website looks like in given template
```

Comment:
I think this is too broad. There are templates that you'll have to be a CSS master to apply.